library/src/main/java/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.java

```diff
@@ -127,13 +129,15 @@
   private transient ScheduledExecutorService scanService;
   private transient AtomicReference<Throwable> threadException;
   protected transient boolean isPolled;
-  protected transient Integer lastPolledRow;
   protected transient LinkedBlockingDeque<T> emitQueue;
   protected transient PreparedStatement ps;
   protected boolean isPollerPartition;
 
   protected transient MutablePair<Integer, Integer> currentWindowRecoveryState;
 
+  private int lastOffset;
+  private int nextOffset;
```

Comment:
I was thinking on the lines if we poller fails while adding the records then we may need to restore the value , but we are assuring the same via variable 'bound'. So will change it.